### ISSUE TITLE
Version 0.3.2 interface deranged

### ISSUE THREAD

**Issue by Abson** [neovim/neovim#8775]
![image](https://user-images.githubusercontent.com/13198014/43138094-6df1cc3e-8f80-11e8-914f-7b5cca704ac4.png)

I found the problem of disordered interfaces after I updated today.

```
NVIM v0.3.2-24-gc8e7a44
Build type: Debug
LuaJIT 2.0.5
Compilation: /usr/bin/cc -Wconversion -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector-strong -fdiagnostics-color=auto -Wno-array-bounds -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/home/parallels/github/neovim/build/config -I/home/parallels/github/neovim/src -I/home/parallels/github/neovim/.deps/usr/include -I/usr/include -I/home/parallels/github/neovim/build/src/nvim/auto -I/home/parallels/github/neovim/build/include
Compiled by root@abson

Features: +acl +iconv +jemalloc +tui 
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/share/nvim"

Run :checkhealth for more info
```

**Comment by Abson** [comment]
The problem will disappear when I scroll the screen in order to change page.

**Comment by bfredl** [comment]
Would you mind try if #8771 fixes the issue?

**Comment by Abson** [comment]
Thx，We want to merge as quickly as possible，let's get rid of this bug.

**Comment by tcbbd** [comment]
Can confirm this is fixed by #8771, as well as the tabline corruption problem.

**Comment by gkapfham** [comment]
Hello @Abson and @tcbbd, I am writing to say that I am also experiencing this issue on Ubuntu 16.04 running `NVIM v0.3.2-dev` installed from the PPA. Lete us know once the fix is available in master and I will try it and report back as to whether or not it solves the glitches that I am seeing on my screen. Thanks to everyone for all of their exceptional work on Neovim &mdash; I use it regularly and really enjoy it!

**Comment by bfredl** [comment]
@gkapfham I just merged a fix, please try latest master.

**Comment by Abson** [comment]
Great, How strong is this community!

**Comment by bfredl** [comment]
Good to know, closing

**Comment by gkapfham** [comment]
Hello @bfredl, I am writing to confirm that I have been using `NVIM v0.3.2-dev` on Ubuntu 16.04 and I have not seen this error again. I should note that, with the previous version that did have the interface glitches, I did not always see the problem immediately. However, I think that I have now used the program for long enough to conclude that the new version does indeed fix the problem. Thanks for this fix, I really appreciate it!
